apps/common-app/src/apps/reanimated/examples/TransformOriginExample.tsx

```diff
@@ -0,0 +1,139 @@
+import { Button, StyleSheet, View } from 'react-native';
+import Animated, {
+  cancelAnimation,
+  Easing,
+  interpolate,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+
+type TransformOriginOption = string | (string | number)[];
+
+export default function TransformOriginExample() {
+  const progress = useSharedValue(0);
+  const currentOrigin = useSharedValue<TransformOriginOption>('center');
+
+  const containerStyle = useAnimatedStyle(() => {
+    const rotation = interpolate(progress.value, [0, 1], [0, 45]);
+    const origin = currentOrigin.value;
+
+    return {
+      transform: [
+        {
+          rotate: `${rotation}deg`,
+        },
+      ],
+      transformOrigin: origin,
+    };
+  });
+
+  const handlePress = (newOrigin: TransformOriginOption) => {
+    currentOrigin.value = newOrigin;
+
+    cancelAnimation(progress);
```

Comment:
Since you're assigning 0 directly to the progress shared value, I think calling `cancelAnimation` is not necessary